trtools/utils/utils.py

```diff
@@ -387,9 +386,9 @@ def GetCanonicalMotif(repseq):
     repseq_r = GetCanonicalOneStrand(ReverseComplement(repseq))
```

Comment:
We also need to change ReverseComplement to properly reverse complement all of the other codes - for instance, the complement of whichever character stands for "A or G" is the character that stands for "T or C".